Singletons/WaveSpecial.gd

```diff
@@ -444,3 +444,7 @@ func _special_applies_to_creep(special: int, creep: Creep) -> bool:
 		special_applies = true
 
 	return special_applies
+
+
+func get_special_buff_map() -> Dictionary:
```

Comment:
Change to
`func get_special_buff(special: int) -> BuffType`
Returning whole list breaks encapsulation, callers will be able to modify the list.

Reply:
Done